presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -71,22 +98,38 @@ Array Functions
 
     Returns the cardinality (size) of the array ``x``.
 
+        SELECT cardinality(ARRAY[10,20,30,40,50],10); -- 5
```

Comment:
nit: remove the second argument (probably a typo? )

Can we make the array size 4 or 6? :)